src/store/index.js

```diff
@@ -17,7 +23,14 @@ const mutations = {
   },
 
   removeNotification(state, notification) {
-    state.notifications.splice(state.notifications.indexOf(notification), 1);
+    actions.deleteNotification(state, notification);
+  },
+
+  removeAllNotifications(state) {
+    state.notifications.forEach((notification) => {
```

Comment:
Any specific reason to use `forEach()`? If not, you can do this:

```javascript
state.notifications = [];
```

Reply:
@sahibalejandro I tried that code but once that line of code was executed I was not able to add new values to the notifications array. Looks like it is something related to losing the reactiveness once I reset the value. I took the code I used from another package.